src/utarray.h

```diff
@@ -92,7 +92,7 @@ typedef struct {
     char *utarray_tmp;                                                        \
     while (((a)->i+(by)) > (a)->n) { (a)->n = ((a)->n ? (2*(a)->n) : 8); }    \
     utarray_tmp=(char*)realloc((a)->d, (a)->n*(a)->icd.sz);                   \
-    if (utarray_tmp == NULL) oom();                                           \
+    if (utarray_tmp == NULL) utarray_oom();                                   \
```

Comment:
Here and above, I would like to see curly braces around the `if` body, as long as we're updating the code.

    if (utarray_tmp == NULL) { \
        utarray_oom();         \
    }                          \